app/views/teams/index.html.erb

```diff
@@ -1,8 +1,8 @@
 <div class="team-cards">
-  <% @teams.each do|team| %>
+  <% @teams.each do |team| %>
   <!-- <a href="<%= team_positions_path(team) %>"> --></a>
     <%= link_to team_positions_path(team) do %>
-      <div class="team-card" >
+      <div class="team-card <%= current_user.nil? ? "team-card-dota" : current_user.show_game == "1" ? "team-card-dota" : "team-card-csgo" %> " style="background-image: url('<%= cl_image_path team.photo.key %>')">
```

Comment:
Did you put a ternary inside a ternary? I'd normally advise against that but it may be cleaner then if elseif, else, in the .erb files.

Reply:
Yes, it is a nested ternary. I would have done if/elsif statements too, but this doesn't fire those pink exceptions with the embedded ruby. Not scalable anyway :)